README.md

```diff
@@ -23,24 +23,19 @@ $ bin/spark-shell --jars spark-csv-assembly-1.0.0.jar
 ```
 
 ## Features
-These examples use a CSV file available for download [here](https://github.com/databricks/spark-csv/raw/master/src/test/resources/cars.csv):
-
-```
-$ wget https://github.com/databricks/spark-csv/raw/master/src/test/resources/cars.csv
-```
+This package allows reading CSV files in local or distributed filesystem as [Spark DataFrames](https://spark.apache.org/docs/1.3.0/sql-programming-guide.html).
+When reading files the API accepts several options:
+* path: location of files. Similar to Spark can be a wildcard.
+* header: when set to true the first line of files will be used to name columns and will not be included in data. All types will be assumed string.
```

Comment:
Oh, I think false is better.  Possibly dropping data is more confusing that having them see their headers in the results.